conda_smithy/configure_feedstock.py

```diff
@@ -1774,7 +1774,9 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         # Specific channel for package can be given with
         #     ${url or channel_alias}::package_name
         # defaults to conda-forge channel_alias
-        "remote_ci_setup": ["conda-forge-ci-setup=3"],
+        # Added py-lief constraint due to current osx-* segfault issues, ref:
+        #     https://github.com/conda-forge/conda-forge.github.io/issues/1823
+        "remote_ci_setup": ["conda-forge-ci-setup=3", "py-lief<0.12"],
```

Comment:
You might need
```suggestion
        "remote_ci_setup": ["conda-forge-ci-setup=3", "'py-lief<0.12'"],
```

Have you checked this?

Reply:
yes, not needed.

Reply:
https://github.com/conda-forge/conda-smithy/blob/c1f48814f72c8e4051f938b69e7116a12115a452/conda_smithy/configure_feedstock.py#L393

Reply:
Do you have an example feedstock with this change?

Reply:
Tested it on Linux-only feedstock before. Now did it on one that has all platforms: https://github.com/conda-forge/libxslt-feedstock/pull/35
On osx there was a faulty log message that I just removed (because it is also not there on other platforms).